lib/src/http.dart

```diff
@@ -212,7 +212,7 @@ class _ThrowingClient extends http.BaseClient {
     }
 
     if (status == 500 &&
-        (request.url.host == 'pub.dartlang.org' ||
+        (request.url.host == 'pub.dev' ||
```

Comment:
Needs to be confirmed that it is OK to change this

Reply:
Again here I think it would be better to accept both the new and old name.